tools/mqppep/mqppep_anova.xml

```diff
@@ -28,47 +50,87 @@
       biocontainer are read-only, so this builds a pseudo-home under /tmp
     -->
     <command detect_errors="exit_code"><![CDATA[
+      (printenv | sort) &&
       cp '$__tool_directory__/mqppep_anova_script.Rmd' . &&
-      cp '$__tool_directory__/mqppep_anova.R'          . &&
+      cp '$__tool_directory__/mqppep_anova.R' . &&
```

Comment:
Interesting, so many scripts. Can you please annotate them via https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-required-files

This will make sure Galaxy knows about them when used in pulsar for example.

Reply:
Done in commit 60de97f